pkg/cli/codeset/register.go

```diff
@@ -53,6 +56,13 @@ LOCATION can be path to local directory or URL of a git repository`,
 	cmd.Flags().StringVarP(&o.Project, "project", "p", "", "the project to which the codeset belongs")
 	cmd.Flags().StringVarP(&o.Description, "desc", "d", "", "codeset description")
 	cmd.Flags().StringSliceVar(&o.Labels, "label", []string{}, "one or more codeset labels associated with the codeset")
+
+	cmd.Flags().StringVarP(&o.Password, "password", "", "", "(FUSEML_PROJECT_PASSWORD) Password of the user accessing a project")
```

Comment:
Here's an idea: shouldn't we automatically set the project and codeset supplied here as the current ones ? I don't fully understand the user and password bits, but I figure that also applies to them.

Reply:
Hm... I'm not sure. Maybe it could be nice.
But it also could be bit confusing, doing something user did not ask for...